src/master-playlist-controller.js

```diff
@@ -7,6 +7,7 @@ import Ranges from './ranges';
 import videojs from 'video.js';
 import AdCueTags from './ad-cue-tags';
 import SyncController from './sync-controller';
+import { translateLegacyCodecs } from 'videojs-contrib-media-sources/es5/codec-utils';
```

Comment:
I think we should plan to not do a deep import in the future, but it is ok for now